spec/generators/dashboard_generator_spec.rb

```diff
@@ -142,7 +142,8 @@ class Shipment < ActiveRecord::Base
           attrs = ShipmentDashboard::ATTRIBUTE_TYPES
 
           expect(attrs[:status]).
-            to eq(Administrate::Field::String.with_options(searchable: false))
+            to eq(Administrate::Field::Enum.with_options(
+              enum: { "pending" => 0, "confirmed" => 1 }))
```

Comment:
Indent the first parameter one step more than `Administrate::Field::Enum.with_options(`.